### ISSUE TITLE
[TBC] Into the Soulgrinder - q.11000

### ISSUE THREAD

**Issue by killerwife** [unified-db/Database#188]
Hello, I have been working on this quest for the past few weeks, and I need help.

The quest has three stages:
1. combat script for Skulloc Soulgrinder summoning - very easy, plenty videos to reproduce it
2. after quest completion script - this is the real problem
   Evidence in form of wowhead comments/memory/partial vids suggest this - a huge audience of Bloodmaul and Bladespire supplicants will spawn (for which there is no sniffed data) and they will either chatter among themselves (this seems to have changed over the years) and play random emotes at each other or kneel at the player. This ties on to Speak with Mog'dorg quest 11022 as well, because it is possible that the kneeling should happen after this quest is completed.
3. the result of the script all over blade's edge mountains - all ogres should become neutral, play conditioned death /say https://github.com/cmangos/issues/issues/886,  that happen only after quest is completed AND also respond to stuff like /silly /kiss /charge etc.

My research so far has completed the 1st part, the death /say and the emotes after quest completion. The problem lies in the "make all ogres in blades edge neutral". I have asked several cmangos developers about any data regarding this afterscript. I talked with Laizerox about it and we suspect a server-side force-faction spell that is cast onto every player in blades edge if the quest is completed (this is just a guess, we need proof). I tried to add "neutral status" for the ogre factions, but they do not contain faction information so this is not the way blizzard did it.

To recap: I need information on the afterscript, what exactly should happen and information on the "make ogres neutral", for reference they are factions 929 and 1001 in faction.dbc

When I have the information and I am happy with the quality of the script, I will pull request it to cmangos.

**Comment by xfurry** [comment]
Try looking around for spells similar to http://www.wowhead.com/spell=22799/king-of-the-gordok 
It might be possible that this spell is server side, but probably unlikely.

Also, I'm not sure about the part:
> we suspect a server-side force-faction spell that is cast onto every player in blades edge if the quest is completed

I think this should apply only to the player that completed the quest and maybe only for a limited time.

**Comment by killerwife** [comment]
I looked through about 5000 spells in QSW, there is no such spell. Oh and the neutral to ogres status is permanent. There is no force-reaction spell related to the two factions of those ogres.

**Comment by xfurry** [comment]
The faction change cannot be on the ogres and permanent...
You realize, that there are millions of players that completed this quest on the retail server, but still the ogre factions are the same even today. :smile: 

Clearly it has to do with a spell, or the information is incorrect or incomplete.

**Comment by killerwife** [comment]
I dont mean on the ogres, I mean that the spell is cast permanently on the players that are currently in blades edge mountains.

**Comment by Grz3s** [comment]
If You remember dk starting zone (final part) player had to go back to capital (stormwind/orgrimmar)...
All npc there had diff faction and they had scripted reaction on player... Im sure that your situation will be simmilar here.
look at this spell ... 58530
also you can read:
https://github.com/unified-db/Database/issues/76

**Comment by killerwife** [comment]
Yes, you are right, all ogres in blades edge mountains have Factions 1780-1785 in faction.dbc, which correspond to 929 and 1001 in factionTemplate.dbc. But the timeline of faction status is different from the DK: Since character creation, up to quest Into the Soulgrinder (not only during), Blade's edge ogres are HOSTILE. Once you finish quest Into the Soulgrinder, ogres are forever NEUTRAL.

It will still be SPELL_AURA_FORCE_REACTION in my opinion, but the conditions when it will be applied will be different.

**Comment by Grz3s** [comment]
fixed some time ago
